### DISCUSSION TITLE
RPI4 + Coral setting

### DISCUSSION THREAD

**Discussion by jvels** [blakeblackshear/frigate#1688] [Ask A Question]
Hi

I have cameras there provide rtsp in 3 res. 1920x1080, 1024x576 and 640x360.

I have enabled:
`detectors:
  coral:
    type: edgetpu
    device: usb`

When I use 1024x576 stream from the camera, I see "real time" pictures in frigate, but if I change to 1920x1080, then there are big delay. Is it because of the object detection I have the delay? If yes, is then possible to use low res. for object detect and record the high res.?

**Comment by lawm** [comment]
You can use a low res stream for detect, and a higher res stream for recording.  See
https://blakeblackshear.github.io/frigate/configuration/cameras#example

**Reply by tlundste** [reply]
I don't have an answer to the poster's question... But, if I understand things, you won't be able to do the low-res detect, high-res record thing in future Frigate versions (0.9.0b) unless you add the different resolutions as different cameras.  Don't put too much effort into the one-camera-definition-to-rule-them-all concept, as I think it is going away.  Please, someone correct me if I'm wrong.

**Reply by lawm** [reply]
That's not true.
I think you read the discussion about someone wanting to 24/7 record low res, and record high res clips (https://github.com/blakeblackshear/frigate/discussions/1502#discussioncomment-1246377) .  That's not supported in 0.9.0.
0.9.0 still supports using low res stream for detect, and can use a high res stream for recording+clips.   Recording+clips must use the same stream now, but detect can use a different stream.

**Reply by tlundste** [reply]
OK.. thank you for clarifying the question, and the answer.
